sw/ground_segment/cockpit/live.ml

```diff
@@ -326,19 +333,6 @@ let mark = fun (geomap:G.widget) ac_id track plugin_frame ->
             let png = sprintf "%s/var/logs/%s" Env.paparazzi_home name in
             GdkPixbuf.save png "png" dest;
             incr i;
-
-      (* Computing the footprint: front_left and back_right *)
```

Comment:
this part is actually not related to the camera footprint but to the "mark" button and screenshot tool. See http://wiki.paparazziuav.org/wiki/GCS#Actions
please add back this code.